### ISSUE TITLE
Oxidized Web Interface Is Not Working: 

### ISSUE THREAD

**Issue by rygumcmxci** [ytti/oxidized#1611]
LibreNMS and Oxidized are installed on the same virtual server; LibreNMS is working fine (only have one device in it at the moment). 

Oxidized as a service is working; starts and pulls the config from the device and I can view the device configs using the "config" tab in the LibreNMS web ui and also in the cli, but the oxidized web interface does not work. 

OS: CentOS 7

Gem List: 

*** LOCAL GEMS ***
asetus (0.3.0)
backports (3.11.4)
bigdecimal (1.2.0)
charlock_holmes (0.7.6)
emk-sinatra-url-for (0.2.1)
ffi (1.9.25)
haml (5.0.4)
htmlentities (4.3.4)
io-console (0.4.2)
json (1.7.7)
multi_json (1.13.1)
net-ssh (4.1.0)
net-telnet (0.1.0)
oxidized (0.24.0)
oxidized-script (0.5.1)
oxidized-web (0.11.1)
psych (2.0.0)
puma (3.11.0)
rack (1.6.10)
rack-protection (1.5.5)
rack-test (0.7.0)
rb-fsevent (0.10.3)
rb-inotify (0.9.10)
rdoc (4.0.0)
rugged (0.27.5)
sass (3.6.0)
sass-listen (4.0.0)
sinatra (1.4.8)
sinatra-contrib (1.4.7)
slop (3.6.0)
temple (0.8.0)
tilt (2.0.8)

Oxidized YAML config: 

https://pastebin.com/MwThhLPQ

Basically everything works besides the oxidized web interface

For reference here is what I used to install oxidized: 

- sudo yum update
- sudo useradd oxidized
- sudo passwd oxidized
- sudo usermod -aG wheel oxidized 
- sudo yum install make cmake which sqlite-devel openssl-devel libssh2-devel ruby gcc ruby-devel libicu-devel gcc-c++
- su – oxidized 
- gem install net-telnet -v 0.1.0
- gem install oxidized
- gem install puma -v '3.11'
- gem install oxidized-script oxidized-web
- OXIDIZED_HOME=/etc/oxidized
- oxidized

**Comment by laf** [comment]
What does 'not work' mean to you? LibreNMS uses the webui it must be running to get the config data.

**Comment by rygumcmxci** [comment]
I am unable to access the oxidzed web interface that should be independent of LibreNMS.

**Comment by laf** [comment]
But LibreNMS uses the web interface to pull in the config from Oxidized so it must be working.

`lsof -i:8888`

Check your firewall as well.

**Comment by rygumcmxci** [comment]
Ok, i'll have to check the firewall to see if anything is going on there. 

The weird thing is I already set this up on a physical server and it is working as expected, but the same configuration does not work on my VM. 

I will update the thread once I have checked the firewall. 

Thanks again.

**Comment by rygumcmxci** [comment]
Ok, so I was finally able to get it working.

There were some minor differences in the firewall setting between my VM and Physical server. 

What I did to resolve the issue was add the following statement: 

firewall-cmd --permanent --zone public --add-port 8888/tcp
